packages/location-field/src/index.tsx

```diff
@@ -735,6 +735,7 @@ const LocationField = ({
       <S.InputGroup>
         {/* location field icon -- also serves as dropdown anchor */}
         <S.Dropdown
+          input={<>{textControl}{clearButton}</>}
```

Comment:
Does that also apply to the mobile view?

Reply:
The mobile view has always had the dropdown in the semantically correct place!